test/versioned/iovalkey-esm/iovalkey.test.js

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import test from 'node:test'
+import assert from 'node:assert'
+import helper from '../../lib/agent_helper.js'
+import params from '../../lib/params.js'
+import urltils from '../../../lib/util/urltils.js'
+import { tspl } from '@matteo.collina/tspl'
+import assertions from '../../lib/custom-assertions/index.js'
+const { assertMetrics } = assertions
+import { removeModules } from '../../lib/cache-buster.js'
+
+// Indicates unique database in Valkey. 0-15 supported.
+const DB_INDEX = 5
+
+test('iovalkey instrumentation', async (t) => {
```

Comment:
you can delete all these tests. we just created ioredis-esm tests for various reasons that are not applicable here.